### PR TITLE
added rle files

### PR DESCRIPTION
**Pull request by Vikram9866**

### CODE REVIEW

**Comment by Vikram9866** [comment]
@cfelton can you review this once ? I made all the necessary changes that you suggested

File: test/.coverage [hunk 1]
```diff
@@ -0,0 +1 @@
... 36, 37, 38, 39, 40, 41, 555, 44, 45, 46, 47, 48, 49, 50, 307, 53, 54, 823, 56, 59, 319, 1088, 1090, 1163, 566, 1094, 437, 74, 1146, 336, 343, 1116, 349, 358, 1127, 360, 1129, 363, 890, 366, 1136, 659, 373, 374, 1144, 377, 378, 1078, 380, 893, 383, 384, 1153, 387, 389, 390, 391, 392, 393, 395, 140, 397, 400, 1170, 403, 793, 415, 417, 1046, 1052, 1233, 1237, 685, 1199, 790, 1267, 30, 696, 442, 699, 956, 189, 702, 799, 962, 195, 197, 201, 205, 462, 1229, 208, 1161, 291, 725, 214, 728, 218, 475, 221, 224, 227, 484, 230, 999, 233, 1002, 236, 451, 239, 1265, 211, 247, 250, 1277, 254, 767], "/usr/local/lib/python3.4/dist-packages/myhdl-1.0dev-py3.4.egg/myhdl/_Simulation.py": [64, 96, 67, 225, 74, 75, 77, 20, 21, 22, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 36, 37, 38, 40, 43, 44, 45, 46, 47, 112, 115, 52], "/usr/local/lib/python3.4/dist-packages/myhdl-1.0dev-py3.4.egg/myhdl/conversion/_verify.py": [1, 2, 3, 4, 5, 6, 7, 8, 10, 12, 13, 14, 15, 16, 18, 20, 22, 24, 27, 28, 29, 31, 33, 36, 37, 39, 41, 43, 44, 45, 46, 47, 48, 51, 52, 53, 54, 55, 56, 59, 60, 61, 62, 63, 64, 65, 66, 69, 70, 71, 72, 73, 74, 75, 76, 80, 81, 82, 83, 84, 87, 88, 89, 90, 91, 92, 96, 98, 100, 101, 102, 104, 246, 247]}}
```
Comment (cfelton) [inline test/.coverage]: don't add the .coverage file

Comment (Vikram9866) [inline test/.coverage]: Done removing it

**Comment by Vikram9866** [comment]
@cfelton. Travis Builder has passed ... I re-opened the PR in the main repo too

**Comment by cfelton** [comment]
@Vikram9866 are you familiar with git rebase?  This branch could use a rebase, many of the intermediate commits (which are good for development) don't contribute much to the history.  If you do an interactive rebase you should be able to squash down some of the inconsequential commit comments (e.g. 'docs').

Here are some links provided by others:
https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase
https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history

See the previous PR discussion:
https://github.com/cfelton/test_jpeg/pull/13

Note, the many commits is what we want for development and GSoC review.  When looking at the history, it will be easier for other to digest if the see a first commit comment line like "initial run-length-encoder (RLE) implementation"  with a much longer explanation in the rest of the commit commit.  Commit comments and history is definitely an art but and important one so that our collection of work is clean.

**Pull request merged**